waiter/src/waiter/marathon.clj

```diff
@@ -209,7 +211,7 @@
                      :intervalSeconds 10
                      :portIndex 0
                      :timeoutSeconds 20
-                     :maxConsecutiveFailures 20}]
+                     :maxConsecutiveFailures 5}]
```

Comment:
Even after the grace period for an unhealthy instance passes, Marathon would wait `maxConsecutiveFailures` x `intervalSeconds` = 200 seconds (3 minutes & 20 seconds) before actually failing and killing the instance. Decreasing `maxConsecutiveFailures` allows us to report back to users more quickly on these instances, and users can just extend their `gracePeriodSeconds` to compensate if they actually need that extra time (which they probably should be doing anyways!). For reference, the default `maxConsecutiveFailures` set by Marathon is 3.